main.py

```diff
@@ -85,10 +85,18 @@ def split_chained_request_name(name):
 
 def get_short_name(name):
     spl = name.split('_')
-    renames = {'b': 'bbbar4l',
-               'ST': 'SingleTop'}
-    return renames.get(spl[0], spl[0])
-
+    short_name = spl
+
+    if(spl == 'b'):
```

Comment:
Have you tried if this works? If you split `b_bbar_4l_TuneCP5_13TeV-powheg-pythia8`, you get a list like this:
```
['b', 'bbar', '4l', 'TuneCP5', '13TeV-powheg-pythia8']
```
Which will never be equal to a string `b`.

Reply:
Also, you don't need parentheses in these if statements.